authentication/handler_test.go

```diff
@@ -553,6 +553,30 @@ var _ = Describe("Handler", func() {
 		}`))
 	})
 
+	It("Allows access token authorization", func() {
+		// Prepare the next handler, which should not be called:
```

Comment:
`s/should not/should/` :stuck_out_tongue:

Reply:
thanks :)